src/cmd/utils.go

```diff
@@ -0,0 +1,92 @@
+package cmd
```

Comment:
We are missing the copyright and license notices. See https://github.com/containers/toolbox/pull/924

Reply:
Good point! Forgot about it.